client/tracks/index.ts

```diff
@@ -0,0 +1,98 @@
+/**
+ * External dependencies
+ */
+import domReady from '@wordpress/dom-ready';
+import { getConfig } from 'wcpay/utils/checkout';
+import { getPaymentRequestData } from 'wcpay/payment-request/utils';
+
+/**
+ * Internal dependencies
+ */
+import events from './events';
+
+declare const window: {
+	wc: {
+		tracks: {
+			recordEvent: (
+				eventName: string,
+				eventProperties: Record< string, unknown >
+			) => void;
+		};
+	};
+	wcTracks: {
+		isEnabled: boolean;
+		recordEvent: (
+			eventName: string,
+			eventProperties: Record< string, unknown >
+		) => void;
+	};
+	wcpaySettings: {
+		testMode: boolean;
+	};
+} & Window;
```

Comment:
We can move window declarations to [globals.d.ts](https://github.com/Automattic/woocommerce-payments/blob/3552d0a91d07ae4ccc862ec59355938b250bafae/client/globals.d.ts). Then you just need to access them directly from global so instead of `window.wcTracks.isEnabled` you write it like `wcTracks.isEnabled`.

Reply:
No issue, we only need to define/mock global in the test's context.